larq_compute_engine/mlir/tf_tfl_passes.cc

```diff
@@ -110,6 +107,7 @@ void AddTFToLCETFLConversionPasses(
   pass_manager->addPass(mlir::TFL::CreateOptimizePass());
   pass_manager->addPass(mlir::TFL::CreateOptimizeLCEPass(
       experimental_enable_bitpacked_activations));
+  pass_manager->addPass(mlir::TFL::CreateLegalizeLCEPass());
```

Comment:
We might need to move this pass all the way to the end to properly handle quantization. I will check that.